src/features/commands/poll.js

```diff
@@ -0,0 +1,145 @@
+                            .setFooter(`Poll ID: ${response.pollId}`);
+
+                        interaction.reply({ content: `Poll "${pollTitle}" has been created successfully on this server.`, ephemeral: true });
+                        interaction.channel.send({ embeds: [publicPollEmbed] });
+                    });
+                });
+                break;
+            case "list":
+                db.getPolls(interaction.guild.id).then(polls => {
+                    if (!polls.length) {
+                        interaction.reply({
+                            content: 'There are no polls in this server.',
+                            ephemeral: true
+                        });
+                        return;
+                    }
+                    const title = polls[0].polls,
+                        listEmbed = new MessageEmbed()
+                            .setColor('#0099ff')
+                            .setTitle('All polls in this server')
+                            .setDescription(title.map((pollTitle) => `⦿ [${pollTitle.split(':')[0]}](https://strawpoll.com/${pollTitle.split(':')[1]})`).join('\n'));
+                    interaction.reply({
+                        embeds: [listEmbed],
+                        ephemeral: true,
+                    })
+                });
+                break;
+            case "results":
+                const pollId = interaction.options.getString("poll-id", true);
+
+                db.getPolls(interaction.guild.id).then(polls => {
+                    if (!polls.length) {
+                        interaction.reply({
+                            content: 'There are no polls in this server.',
+                            ephemeral: true
+                        });
+                        return;
+                    }
+                    strawpoll.getStrawpollResults(pollId).then(resp => {
+                        const quickChartClient = new QuickChart();
+                        let answers = [];
+                        let votes = [];
+                        let stringEmbed = '';
+
+                        for (let i = 0; i < resp.pollAnswersArray.length; i++) {
+                            answers.push(resp.pollAnswersArray[i].answer);
+                            votes.push(resp.pollAnswersArray[i].votes);
+                            stringEmbed += `Votes for choice \`${resp.pollAnswersArray[i].answer}\`: ${resp.pollAnswersArray[i].votes}\n`;
+                        }
+
+                        quickChartClient.setConfig({
+                            type: 'bar',
+                            data: { labels: answers, datasets: [{ label: 'Votes', data: votes }] },
+                        });
+
+                        quickChartClient.setBackgroundColor("#ffffff");
+
+                        const resultsEmbed = new MessageEmbed()
+                            .setColor("#0099ff")
+                            .setTitle("Poll results")
```

Comment:
Would be nice to include name of Poll initiator here as I saw that in DB you're collecting who's creating the polls, so it would look like something like _"Jaska's poll results"_